jekyll/_cci2/faq.md

```diff
@@ -1,41 +1,69 @@
 ---
 layout: classic-docs
-title: "Frequently Asked Questions (FAQ)"
-short-title: "FAQ"
-description: "Frequently asked questions about CircleCI 2.0"
-categories: [getting-started]
-order: 50
+title: "Migration FAQ"
+short-title: "Migration FAQ"
+description: "Frequently asked questions about CircleCI 2.0 Migration"
+categories: [migration]
+order: 1
 ---
 
+* Contents
+{:toc}
+
+## Why Migrate from CircleCI 1.0 to 2.0?
+
+- CircleCI 2.0 includes a significant rewrite of container utilization to run more jobs faster and to prevent available containers from sitting idle. 
+- In 2.0, Jobs are broken into Steps. You compose these Steps within a Job at your discretion, giving you greater flexibility to run your build the way you want. 
+- 2.0 Jobs support almost all public Docker images. You can also create a custom image and run jobs on it or compose multiple images together (like MySQL 5.7 + Redis 3.2) and run jobs on them as if they were a single image.
```

Comment:
Combining multiple images in Docker wouldn't be recommended for an example like this. The example suggests combining two database images but you wouldn't do this. You would keep them separate and call them both as 'service' containers in the build.

e.g. `.circleci/config.yml` might look like this:

```
jobs:
  build:
    working_directory: ~/main
    docker:
      - image: python:3.6.0
      - image: mysql:X.Y.Z
      - image: redis:X.Y.Z
```

Suggest change to:

> 2.0 Jobs support almost all public Docker images. You can also create a custom image with your own dependencies specified (see: https://circleci.com/docs/2.0/custom-images/)

With 'multi stage builds' (https://docs.docker.com/engine/userguide/eng-image/multistage-build/#use-multi-stage-builds) this recommendation can be changed a bit. But that's not release yet.